ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -134,12 +165,29 @@
         {
             "cell_type": "code",
             "source": [
-                "public IfrsVariable CreateVariable(IDataRow row, Guid partition)",
+                "public IfrsVariable CreateVariable(IDataRow row, Guid partition, bool isProjectionTest)",
```

Comment:
imo you don't need this extra input and always try to parse the extra Values

Reply:
This boolean is not there anymore.